python/BeamSpotScript.py

```diff
@@ -274,24 +279,34 @@ def make_pdfs(tag='',era='', useMCtruth=False):
     print 'make_pdfs()...'
 
     zbins, zmin, zmax = 55, -15., 15.
-
-    mc = ROOT.TFile('./NanoAOD.root','READ')
-    tmc = mc.Get('demo/Events')
-    hmc = ROOT.TH1F('mc_vtxZ','', zbins, zmin, zmax)
-    tmc.Draw('GenVertex_z>>mc_vtxZ') 
     fitmc = ROOT.TF1('fitmc', '[0]/TMath::Sqrt(2*TMath::Pi())/[1]*TMath::Exp( -0.5*(x-[2])*(x-[2])/[1]/[1] )', zmin, zmax)    
-    fitmc.SetParameter(0, hmc.Integral())
-    fitmc.SetParameter(1, hmc.GetRMS())
-    fitmc.SetParameter(2, hmc.GetMean())
-    hmc.Fit(fitmc, 'R')
-    fitmc.SetParameter(0, 1.0)
-    hmc.Scale(1./hmc.Integral())
```

Comment:
I would keep this commented.

Reply:
Yes, but for this we need the test root file.

Reply:
not if you keep it commented.